main.py

```diff
@@ -0,0 +1,72 @@
+from cart import Cart
+from product import Product
+if __name__ == "__main__":
+    cart = Cart()
+    apple = Product("apple", 0.5)
+    bread = Product("bread", 6)
+    water = Product("water", 2)
+    orange = Product("orange", 1)
+## jak wygląda sklep?roduct
+## 1.wchodzę witam się 2.mówię sprzedawcy co potrzebuję 3. gdy mam wszystko płace 4. pakuję to do siatki żegnam się i ide
+    print(input("say hello: "))
```

Comment:
Do not repeat yourself. What u could do instead repeating the code?

Reply:
done